kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/facility-permissions-form/index.vue

```diff
@@ -0,0 +1,223 @@
+<template>
+
+  <div>
+
+    <!-- Cannot match spec with current core modal -->
+    <!-- Should be its own component -->
+    <core-modal
+      :title=" $tr('facilityPermissionsPresetDetailsHeader')"
+      @cancel="hideFacilityPermissionsDetails"
+      :enableBgClickCancel="true"
+      v-if="permissionPresetDetailsModalShown">
+
+      <dl class="permission-preset-human">
```

Comment:
The text looks kind of crowded in the UI.

Reply:
talked to @jtamiace a bit more about it yesterday, and she OK'd a new line height. We can talk about something a little taller if others agree?